Sources/Overlays/_Testing_CoreGraphics/Attachments/_AttachableImageContainer.swift

```diff
@@ -0,0 +1,163 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+#if SWT_TARGET_OS_APPLE && canImport(CoreGraphics)
+@_spi(Experimental) public import Testing
+private import CoreGraphics
+
+private import ImageIO
+import UniformTypeIdentifiers
+
+/// ## Why can't images directly conform to Attachable?
```

Comment:
Silly question, but is there any benefit to this being a `///` (doc comment) instead of `//` since it's not attached to any declaration? Very useful comment, regardless

Reply:
Just for the formatting.